R/ABSLOPE.R

```diff
@@ -0,0 +1,350 @@
+#Formated with tidyverse
+library(SLOPE)
+library(truncdist)
+library(nlshrink)
+library(MASS)
+library(glmnet)
+library(missMDA)
+library(mice)
+# library(matlib)
+
+ABSLOPE <- function(X, y, lambda, a = 1, b = 10, beta.start = NA, maxit = 300, case = "MCAR", seed = NA, print_iter = FALSE, tol_em = 1e-6, impute = "PCA", sigma.known = NA, sigma.init = NA, scale = FALSE, method_na = "lineq") {
```

Comment:
Please document your function with roxygen documentation (see the documentation for SLOPE() for instance) and avoid these long lines (break at 80 character)